.gitignore

```diff
@@ -109,3 +109,5 @@ tests/integration/data
 
 # setuptools_scm generated version file
 bentoml/_version.py
+typings/pandas
+typings/numpy
```

Comment:
we will need to add all the other libs in the list below?

maybe let's ignore everything under `typings` here?  and when we actually need to commit some stub files to the repo, we can manually do that.

Reply:
sure

Reply:
done